DXRando/DeusEx/Classes/SmugglerElevatorTracker.uc

```diff
@@ -0,0 +1,49 @@
+class SmugglerElevatorTracker expands FlagToggleTrigger;
+
+var bool street, elevatorMoved;
+
+static function SmugglerElevatorTracker CreateSET(DXRando dxr)
+{
+    local SmugglerElevatorTracker set;
+
+    set = dxr.Spawn(class'SmugglerElevatorTracker',, 'elevatorbutton');
+    set.street = dxr.localURL == "02_NYC_STREET" || dxr.localURL == "04_NYC_STREET" || dxr.localURL == "08_NYC_STREET";
```

Comment:
I still don't like these maps being hard-coded in the trigger itself, even if it's in a static function.  I think a book street should just be passed into the Create function, populated by the call in Fixup.  Map-specifics stay out of the logic that way, and stay with the rest of the map-specifics information

Reply:
The idea behind this class is that it's very specific to vanilla maps. That's the point, to offload what shouldn't be done in `FlagToggleTrigger` to somewhere else so it doesn't have to be duplicated somewhere else 3 to 6 times. That's why adding these is hidden behind `#var(vanilla)` checks. Specifics should be left out, because that's against the point.